.github/workflows/test.yml

```
@@ -0,0 +1,350 @@
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+          cache: 'pip'
+      
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: 'npm'
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)